fission-server/config/settings.toml

```diff
@@ -1,3 +1,6 @@
+[database]
+url = "postgres://postgres:postgres@localhost:5432/fission-server"
```

Comment:
At some point it could be great to support running sqlite locally, to avoid having a full-blown postgres instance to manage. I think diesel supports sqlite, @zeeshanlakhani do you have examples of how to set up dev/prod configs with our template?

Reply:
diesel does support sqlite. In terms of configuration, conceptually this is probably fine. Of course this should also be associated in the Rust settings.toml/config. Here's an example from the past (working w/ Dynamo):

```rust
#[serde_as]
#[derive(Debug, Deserialize)]
pub struct Dynamo {
    pub event_ttl: u64,
    #[serde_as(as = "Option<DisplayFromStr>")]
    #[serde(default)]
    pub url: Option<Uri>,
}
```

Reply:
Oh of note @blaine @icidasset, diesel-async does not support sqlite however, diesel itself does. Something to think about / be aware of.

Reply:
I'm less hung up on sqlite specifically than some lightweight in-memory option. Sqlite would be nice because it's _super trivial_ to both store data between runs and blow away test databases. I guess postgres isn't that much more complexity (especially with nix?). My main goal would be to have something that's zero-config and reproducible for devs to run a local instance, so I'll support anything that achieves that goal!